src/applications/check-in/api/local-mock-api/phase.3.js

```diff
@@ -9,6 +9,8 @@ const featureToggles = require('./mocks/feature.toggles');
 const delay = require('mocker-api/lib/delay');
 
 let hasBeenValidated = false;
+const retries = { current: 0, max: 3 };
+const mockUser = { last4: '5555', lastName: 'lastName' };
```

Comment:
Everything looks good so far. Does this cause this combination to be the only way to get past the validate page? 

If so , can you add that to the README?

Reply:
Good point - that is the case unfortunately. I can update the readme if this is the approach we end up going with.